src/test/java/com/string/UpperTest.java

```diff
@@ -8,8 +8,8 @@
     @Test
     void testUpper() {
         Assertions.assertEquals(toUpperCase("abc"), ("abc").toUpperCase(), "The strings are equals");
```

Comment:
Instead of "The strings are equals" I think it should be "The strings are equal"

Reply:
I'll change it an make a new pull request. :)